streamalert_cli/terraform/kinesis_streams.py

```diff
@@ -35,10 +36,11 @@ def generate_kinesis_streams(cluster_name, cluster_dict, config):
         '{}_{}_streamalert'.format(prefix, cluster_name)
     )
 
-    cluster_dict['module']['kinesis_{}'.format(cluster_name)] = {
+    module_name = 'kinesis_{}'.format(cluster_name)
+    cluster_dict['module'][module_name] = {
         'source': './modules/tf_kinesis_streams',
         'account_id': config['global']['account']['aws_account_id'],
-        'region': config['clusters'][cluster_name]['region'],
+        'region': kinesis_module.get('region', config['global']['account']['region']),
```

Comment:
This change will force all kinesis to be in the same region, although kinesis is per cluster basis. I think we should have `region` setting in cluster conf files, under `kinesis` module.

Reply:
that is what this does now - I will update the docs to reflect the region setting in the kinesis module

Reply:
I've pushed a commit to address this and remove the region setting from this module. as discussed, this will actually break the lambda event source mapping